mkosi.build.chroot

```diff
@@ -16,6 +16,7 @@ TEMP=$( \
     --long 'fio' \
     -n 'mkosi.build' \
     -- "$@" \
+    || true
```

Comment:
This is required because `getopt` otherwise freaks out when this build script is run with the clangd flags. I think many of these flags are no longer necessary since the split into multiple recipes so maybe it would be easier to just remove this altogether?

Reply:
We could leave this file and the main build file alone if it were possible to reset the build script list and only add a single one. However, currently this seems impossible from the cmdline which I opened https://github.com/systemd/mkosi/issues/3208 for